modules/layers/package.json

```diff
@@ -69,9 +68,14 @@
     "@turf/transform-translate": ">=4.0.0",
     "@turf/union": ">=4.0.0",
     "cubic-hermite-spline": "^1.0.1",
-    "deck.gl": ">=6.0.5",
     "geojson-types": "^2.0.1",
     "global": ">=4.3.0",
     "viewport-mercator-project": ">=5.0.0"
+  },
+  "peerDependencies": {
```

Comment:
@supersonicclay Should we keep supporting `>=6.0.5` or switch to `^7.0.0`? What do you think?

Reply:
I would not recommend supporting older versions of deck.gl. It is of course doable but the test matrix becomes big and it could be costly to debug breakages.

At this stage in nebula.gl's "lifecycle" (just about to publish nebula.gl 1.0) I don't see supporting deck.gl v6 adding much value to users.